libs/common/src/types/mongo-db-models/tfm-facilities.ts

```diff
@@ -47,7 +47,7 @@ interface BaseAmendment {
   createdAt: UnixTimestamp;
   updatedAt: UnixTimestamp;
   status: AmendmentStatus;
-  version: number;
```

Comment:
Out of interest, do you know what version does / when it will be present?

Reply:
version is just the number of amendments that have made, but I think implementing it on portal as it has been done on tfm will be quite complicated - I think we'll need to think about it when we work out how to maintain TFM amendments existing functionality (and maybe its worth just ignoring `version` altogether? I dont know where its used)

Reply:
dtfs-central-api\src\v1\controllers\tfm\amendments\tfm-post-amendments.controller.ts is where its initialised

Reply:
Good research, agreed it's not a used field that I'm aware of - so makes sense to drop it portal side.
Do you think there's a case for removing it from `BaseAmendment` and having it as a required field in `TfmFacilityAmendment`?

Reply:
Have done now, had to update a repo function a bit but thats definitely for the best anwyay.

Reply:
Amazing, thanks for doing that!